.github/CONTRIBUTING.md

```diff
@@ -169,12 +169,32 @@ See [Contributing to documentation related to PowerShell](#contributing-to-docum
 
 #### Pull Request - Code Review
 
-* After a successful test pass,
-  the area maintainers will do a code review,
+* Roles of a PR: Author, Reviewer, and Assignee
```

Comment:
Title should be:

Roles and Responsibilities of a PR: Author, Reviewer, and Assignee